cfme/tests/configure/test_default_views_cloud.py

```diff
@@ -41,7 +41,11 @@ def test_default_view_cloud_reset(appliance):
     assert not view.tabs.default_views.reset.disabled
 
 
-def set_and_test_default_view(appliance, group_name, expected_view, page):
+@pytest.mark.parametrize('group_name', gtl_params.keys())
+@pytest.mark.parametrize('expected_view', ['List View', 'Tile View', 'Grid View'])
+def test_default_view(appliance, group_name, expected_view):
```

Comment:
I like the changes that you've made here @valaparthvi, but I think that we should maintain a bit more complexity in the test name in order to make sure we've got unique test function names across this module and `test_default_views_infra.py` in the same test folder.

The `infra` targeted module is written in just the same way, and could benefit from the same changes to parametrization and naming.

I'd suggest we use a name that still includes the provider type, like `test_default_cloud_view`, `test_compare_cloud_view`.

Reply:
I am working on exactly the same changes that you've mentioned @mshriver. I'll update soon.